xds/internal/xdsclient/client_new.go

```diff
@@ -57,11 +55,7 @@ const NameForServer = "#server"
 // only when all references are released, and it is safe for the caller to
 // invoke this close function multiple times.
 func New(name string) (XDSClient, func(), error) {
```

Comment:
I don't see a reason for any of these function in this file to exist now. All xDS clients must live in the context of a pool. I see only two non-test references to this `New`, and they are from the xDS resolver and the xDS server. These callsites should be changed to use the pool that gets passed in through the dial option or the server option respectively. And no pool was passed in through the options, then it should use the default pool. That means that we need to export the default pool.

Reply:
All removed and exists only in pool now.